src/stretchy.js

```diff
@@ -0,0 +1,179 @@
+/**
+ * This file provides support to buildMathML.js and buildHTML.js
+ * for stretchy wide elements rendered from background-image SVG files
+ * and other CSS trickery.
+ */
+
+const buildCommon = require("./buildCommon");
+const mathMLTree = require("./mathMLTree");
+const utils = require("./utils");
+
+const stretchyCodePoint = {
+    widehat : "^",
+    widetilde : "~",
+    undertilde : "~",
+    overleftarrow : "\u2190",
+    underleftarrow : "\u2190",
+    xleftarrow : "\u2190",
+    overrightarrow: "\u2192",
+    underrightarrow: "\u2192",
+    xrightarrow : "\u2192",
+    overbracket : "\u23b4",
+    underbrace : "\u23b5",
+    overbrace : "\u23de",
+    underbracket : "\u23df",
+    overleftrightarrow : "\u2194",
+    underleftrightarrow : "\u2194",
+    xleftrightarrow : "\u2194",
+    Overrightarrow : "\u21d2",
+    xRightarrow : "\u21d2",
+    overleftharpoon : "\u21bc",
+    xleftharpoonup : "\u21bc",
+    overrightharpoon : "\u21c0",
+    xrightharpoonup : "\u21c0",
+    xLeftarrow : "\u21d0",
+    xLeftrightarrow : "\u21d4",
+    xhookleftarrow : "\u21a9",
+    xhookrightarrow : "\u21aa",
+    xmapsto : "\u21a6",
+    xrightharpoondown : "\u21c1",
+    xleftharpoondown : "\u21bd",
+    xrightleftharpoons : "\u21cc",
+    xleftrightharpoons : "\u21cb",
+    xtwoheadleftarrow : "\u219e",
+    xtwoheadrightarrow : "\u21a0",
+    xLongequal : "=",
+    xtofrom : "\u21c4",
+};
+
+const mathMLnode = function(label) {
+    const node = new mathMLTree.MathNode(
+        "mo", [new mathMLTree.TextNode(stretchyCodePoint[label.substr(1)])]);
```

Comment:
The mix of strings starting with and without `\\` is confusing.  I kind of wish we removed the `\\` after parsing so we didn't have to deal with this here and there.  No action action required.  Just thinking about how we can make the code base easier to work with in the future.